request-type.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Request } from 'express'
```

Comment:
what is it?

Reply:
request type from express
I found this in the documentation for nestJS